backend/pgnService/game/update.ts

```diff
@@ -144,60 +129,81 @@ function getRequest(event: APIGatewayProxyEventV2): UpdateGameRequest {
  * @param request The UpdateGameRequest to process.
  * @returns A GameUpdate based on the given request.
  */
-async function getGameUpdate(
-    request: UpdateGameRequest,
-): Promise<[GameUpdate | null, GameImportHeaders | null]> {
+async function getGameUpdate(request: UpdateGameRequest): Promise<GameUpdate> {
     const update: GameUpdate = {
         updatedAt: new Date().toISOString(),
     };
 
-    if (request.orientation) {
-        update.orientation = request.orientation;
-    }
     if (request.unlisted !== undefined) {
         update.unlisted = request.unlisted;
-        update.publishedAt = request.unlisted ? null : new Date().toISOString();
-        update.timelineId = request.unlisted
-            ? ''
-            : `${update.publishedAt?.split('T')[0]}_${uuidv4()}`;
+
+        if (request.unlisted) {
+            update.publishedAt = null;
+            update.timelineId = '';
+        } else {
+            update.publishedAt = new Date().toISOString();
+            update.timelineId = `${update.publishedAt?.split('T')[0]}_${uuidv4()}`;
+        }
+    }
+
+    if (request.orientation) {
+        update.orientation = request.orientation;
     }
 
+    let result: string | undefined = request.headers?.result;
```

Comment:
Can we move `result` inside the if since it's not used outside it?

Reply:
Whoops. Looks like old code.